src/index.js

```diff
@@ -28,7 +28,7 @@ export { default as Identifier } from './Identifier/Identifier';
 export { default as Image } from './Image/Image';
 export { default as InlineHelp } from './InlineHelp/InlineHelp';
 export { default as InputGroup } from './InputGroup/InputGroup';
-export { default as FormGroup } from './InputGroup/FormGroup';
+export { default as FormGroup } from './Forms/FormGroup';
```

Comment:
Probably worth reordering this list by alpha - it will put `FormGroup` up by the other `Form*` exports.